.pre-commit-config.yaml

```diff
@@ -22,13 +22,11 @@ repos:
       - id: trailing-whitespace
       - id: end-of-file-fixer
 
-  - repo: local
+  - repo: https://github.com/PyCQA/isort.git
+    rev: 5.7.0
     hooks:
       - id: isort
-        name: isort
-        entry: python -m isort
         args: [--settings-path, ./pyproject.toml]
-        language: system
         types: [python]
```

Comment:
this probably can be removed too, can you check?

Reply:
applied